docs/api/version-history.md

```diff
@@ -20,6 +20,14 @@ keywords: "API, Docker, rcli, REST, documentation"
 * The `VirtualSize` field in the `GET /images/{name}/json`, `GET /images/json`,
   and `GET /system/df` responses is now omitted. Use the `Size` field instead,
   which contains the same information.
+* Deprecated: The `is_automated` field in the `GET /images/search` response has
+  been deprecated and will always be set to false in the future because Docker
+  Hub is deprecating the `is_automated` field in its search API. The deprecation
+  is_ not versioned, and applies to all API versions.
```

Comment:
A rogue underscore snuck in. Hope it doesn't mess up the markdown rendering too much

Reply:
Ah, derp! ❤️ 😂 

Looks like GitHub is fine showing it as-is `s_`, but now I can't un-see; I'll fix it in post 😂 

<img width="377" alt="Screenshot 2023-08-02 at 09 08 42" src="https://github.com/moby/moby/assets/1804568/0825d0f1-f1e6-40b6-a8cf-2d058539290d">

Reply:
- Here goes: https://github.com/moby/moby/pull/46143